src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -737,7 +737,9 @@ private void ScrollViewer_ScrollChanged(object sender, ScrollChangedEventArgs e)
             {
                 var first = _scrollViewer.VerticalOffset;
                 var last = _scrollViewer.ViewportHeight + first;
-                if (_scrollViewer.ViewportHeight > 0 && last >= Items.Count)
+
+                // Minus one to account for the loading indicator
+                if (_scrollViewer.ViewportHeight > 0 && last >= Items.Count - 1)
```

Comment:
Is this indicator always going to be there?

Can we change the code so that we're not using an indicator that's added to the item list potentially? Maybe a state that's independent from the list of items we're going to show to the user?

Reply:
> Can we change the code so that we're not using an indicator that's added to the item list potentially?

Tried that before and ended up having issues with scrolling (couldn't do item-by-item scrolling). I can try figuring another strategy that avoids that issue, but I'm not sure it's in scope for fixing this bug.

Reply:
Coming back to this question - Is this indicator always going to be there? If yes, let's merge now and follow up on a better solution later. I'd like this fix considered for servicing.